src/cudamatrix/cu-kernels-ansi.h

```diff
@@ -30,6 +30,9 @@
 #include "cudamatrix/cu-matrixdim.h"
 
 #if HAVE_CUDA == 1
```

Comment:
while if I remove this header <cublas_v2.h>, an identifier error occurs: 
../cudamatrix/cu-kernels-ansi.h(102): error: identifier "cublasHandle_t" is undefined

../cudamatrix/cu-kernels-ansi.h(104): error: identifier "cublasHandle_t" is undefined

cu-kernels.cu(746): error: identifier "cublasHandle_t" is undefined

cu-kernels.cu(2360): error: identifier "cublasHandle_t" is undefined

cu-kernels.cu(2364): error: identifier "cublasHandle_t" is undefined

5 errors detected in the compilation of "/tmp/tmpxft_00000df0_00000000-19_cu-kernels.compute_53.cpp1.ii".